crates/viewer/re_selection_panel/src/visualizer_ui.rs

```diff
@@ -216,9 +217,15 @@ fn visualizer_components(
         let result_default = query_result.defaults.get(component_descr);
         let raw_default = non_empty_component_batch_raw(result_default, component_descr);
 
-        let raw_fallback = visualizer
-            .fallback_provider()
-            .fallback_for(&query_ctx, component_descr.component_name);
+        // If we don't have a component name, we don't have a way to retrieve a fallback. Therefore, we return a `NullArray` as a dummy.
```

Comment:
maybe better to push that down into `fallback_for`
will be easier when we then later refactor fallback system to be a registry of simple descriptor rules (I still very much intend to go there :))